tests/test_transactions_api.py

```diff
@@ -303,7 +304,7 @@ def test_column_formatting(self) -> None:
         assert response.status_code == 200, response.data
         assert len(data["data"]) == 1
         assert data["data"][0]["event_id"] == first_event_id
-        assert data["data"][0]["span_id"] == "8841662216cc598b"
+        assert data["data"][0]["span_id"] == "841662216cc598b1"
```

Comment:
could we maybe extract this to a constant, to make it more straightforward if we want to update it again?

Reply:
Since the number actually changes (this happens to be the first) it's not possible to make it a true constant.